daffodil-sapi/src/test/scala/org/apache/daffodil/example/TestScalaAPI.scala

```diff
@@ -1207,6 +1207,26 @@ class TestScalaAPI {
     assertEquals(expectedData, bos.toString())
   }
 
+  @Test
+  def testScalaAPI26(): Unit = {
+    val c = Daffodil.compiler()
+    val schemaFile = getResource("/test/sapi/mySchema6.dfdl.xsd")
+    val pf = c.compileFile(schemaFile)
+    assertTrue(pf.isError())
+    try {
+      pf.onPath("/")
+    } catch {
+      case e: IllegalStateException => {
+        val msg = e.getMessage
+        val cause = e.getCause.toString
+        assertTrue(msg.contains("ProcessorFactory onPath() must not"))
```

Comment:
These couple lines checking the message... I wouldn't bother with these first two checks. Just that you got Schema Definition Error and "nonExistant" is enough.